packages/iris-grid/src/IrisGridRenderer.ts

```diff
@@ -301,6 +301,19 @@ class IrisGridRenderer extends GridRenderer {
       (scrimBlurSize * 2 + (height - gridY)) * loadingScrimProgress
     );
 
+    // add an overlay of the scrim color to darken things a bit
+    context.globalCompositeOperation = 'overlay';
+    if (scrimColor != null) {
+      context.fillStyle = scrimColor;
+    }
```

Comment:
I don't think this is needed again, as it's already set above

Reply:
right.